src/migrations/20210930173153-add-grantee-type-to-grantee.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
Do up & down need to return something? Only ask because that was the change we had to make to adding the program specialist, etc to grants

Reply:
Oh yeah, I see what you are saying now. I think the difference here is we aren't running inside a transaction. The transaction call returns a promise, which is then waited on by the caller of the migration (because the code was updated to return the transaction). Another way to solve the issue in #423 (I _think_) would have been to `await`ed the transaction call.